resources/user-agents/browsers/safari/safari-15-0.json

```diff
@@ -0,0 +1,33 @@
+{
+    "division": "Safari #MAJORVER#.#MINORVER#",
+    "versions": ["15.0", 15],
```

Comment:
These two versions are the same, as the Zero will be added automatically, if a `#MINORVER#` placeholder is there.